### PR TITLE
update image data repository link

### PR DESCRIPTION
**Pull request by alpers**

### CODE REVIEW

File: image-search-datasets.md [hunk 1]
```diff
@@ -92,7 +92,7 @@ This is a dataset of images which have been pulled from a collection of digitise
...
+There have also been projects to tag the dataset [using machine learning](https://blogs.bl.uk/digital-scholarship/2016/11/sherlocknet-update-millions-of-tags-and-thousands-of-captions-added-to-the-bl-flickr-images.html). This work makes it possible to search by tags, but we might want a 'richer' ability to search. For this particular experiment, we'll work with a subset of the collections which contain "embellishments". This dataset is a bit smaller, so it will be better for experimenting with. We can get the full data from the British Library's data repository: [https://doi.org/10.21250/db17](https://doi.org/10.21250/db17). Since the full dataset is still fairly large, you'll probably want to start with a smaller sample.
```
Comment (osanseviero) [inline image-search-datasets.md:95]: I think this was intended to be http://hf.co/blog/image-search-datasets

Comment (alpers) [reply image-search-datasets.md:95]: I can change it to "http://hf.co/blog/image-search-datasets" but are you sure it should be that?
There is a shared dataset in the link "https://doi.org/10.21250/db17"

Comment (osanseviero) [reply image-search-datasets.md:95]: Ah sorry, you're right!

**Review by osanseviero — APPROVED** [review]
Thank you so so much for the fix! 🚀

**Pull request merged**